Operational-Procedures/Scheduler/startup/index.rst

```diff
@@ -0,0 +1,207 @@
+For an overview of the full set of parameters see the `Scheduler CSC configuration documentation`_.
+
+.. _Scheduler CSC configuration documentation: https://ts-scheduler.lsst.io/configuration/configuration.html
+
+.. _initializing-the-scheduler-csc-determining-configuration:
+
+Determining configuration
+=========================
+
+For now it suffices to say that, before enabling the CSC, the user must know in advance which configuration should be used for the night.
+During commmissioning it is likely that we will have more than one suitable configurations for every single night, that prioritizes different surveys that can be selected dependending on the conditions.
+These will, in general, be in the form of a configuration label.
+
+The available labels (and associated configuration files) can be found in the Scheduler configuration directory of the `OCS configuration repository`_.
+This information is also published by the CSC in the ``settingVersions`` event, which can be inspected via SAL from a notebook in nublado or in the EFD from chronograf.
+
+.. _OCS configuration repository: https://github.com/lsst-ts/ts_config_ocs
+
+From a notebook one would do something like:
+
+.. code:: ipython3
+
+    import logging
+
+    from lsst.ts import salobj
+
+.. code:: ipython3
+
+    logging.basicConfig(level=logging.DEBUG)
+
+.. code:: ipython3
+
+    domain = salobj.Domain()
+
+Below, note the use of ``index=2``, which means AT Scheduler.
+For MT we would use ``index=1``.
+
+.. code:: ipython3
+
+    remote = salobj.Remote(domain, "Scheduler", index=2)
+
+.. code:: ipython3
+
+    await remote.start_task
+
+.. code:: ipython3
+
+    setting_versions = await remote.evt_settingVersions.aget(timeout=5)
+
+.. code:: ipython3
+
+    print(setting_versions.recommendedSettingsLabels)
+
+From chronograf, the following query would also give you the :ref:`latest available labels <fig-chronograf-scheduler-labels>` (again, note we are explicitly specifying AT Scheduler by selecting ``"SchedulerID" = 2``):
+
+.. code-block:: text
+
+    SELECT "recommendedSettingsLabels" FROM "efd"."autogen"."lsst.sal.Scheduler.logevent_settingVersions"  WHERE "SchedulerID" = 2 ORDER BY DESC LIMIT 1
+
+.. figure:: ./_static/chronograf_scheduler_labels.png
+    :name: fig-chronograf-scheduler-labels
+
+    Latest AT Scheduler setting versions labels from chronograf
+
+.. _initializing-the-scheduler-csc-setting-csc-log-level:
+
+Setting CSC Log Level
+=====================
+
+Before starting, it is recommended to set the Scheduler logging level to ``DEBUG``.
+This can be done from the using the `ATQueue`_ LOVE interface by executing the SAL Script ``run_command.py`` with the following configuration:
+
+.. code-block:: text
+
+    component: Scheduler:2
+    cmd: setLogLevel
+    parameters:
+        level: 10
+
+.. figure:: ./_static/atqueue-run-cmd-scheduler-set-log-level.png
```

Comment:
in this figure, you show level: 10

However, in the screenshot you show it as "warning" in the dropdown. This is for the script, but it's a bit confusing.

You mentioned in the training that the scripts will be sent in debug level IF the CSC is set to debug level. So I think that makes the drop-down invalid?

Besides adding some text here to capture that, I wonder if it's worth changing the label in the LOVE to say "Script Log Level" instead of just "Log Level"

Reply:
So that box only affects Scripts send through LOVE.. The Scheduler matches its own log level with the Script it sends to the SQ.. But yeah, probably a good idea to clarify that in the LOVE view.

Reply:
I notice all of the screenshots display "warning" but I always choose default, and then filter the log messages later.